registry.json

```diff
@@ -1,3 +1,5 @@
 {
-  "dockstore" : "https://www.dockstore.org:8443"
+  "dockstore" : [
+      "https://www.dockstore.org:8443/api/v1/metadata"
+    ]
```

Comment:
It's so a given site (like Dockstore) can advertise multiple versions of the API if need be.  The metadata endpoint includes the API version ID.